### PR TITLE
Towards 0.6.7.

### PR DESCRIPTION
**Pull request by sjrd**
For reference, the Towards 0.6.6 commit was d0f87774796bc7d9d79a97acfedfa0139cd7ea09

### CODE REVIEW

**Comment by sjrd** [comment]
Review by @nicolasstucki for fast-forward push to master (not through GitHub's merge button).

File: project/Build.scala [hunk 1]
```diff
@@ -98,10 +98,12 @@ object Build extends sbt.Build {
           case ScalaJSCrossVersion.binary => previousBinaryCrossVersion
           case crossVersion               => crossVersion
         }
+        val prevExtraAttributes =
+          thisProjectID.extraAttributes.filterKeys(_ != "e:info.apiURL")
```
Comment (sjrd) [inline project/Build.scala]: This is actually a fix for the issue I experienced in #2019, which came back hard on this commit for all Scala versions, not just 2.12.x.

Comment (nicolasstucki) [inline project/Build.scala]: Then I suggest adding a comment that mentions why we need to filter that key.

**Comment by nicolasstucki** [comment]
That is all, otherwise LGTM

**Comment by sjrd** [comment]
Updated.

**Comment by nicolasstucki** [comment]
LGTM

**Pull request merged**